web/viewer.css

```diff
@@ -1264,6 +1223,49 @@ dialog :link {
 }
 
 #secondaryToolbar {
+  .toolbarButton {
+    border-radius: 0;
+    display: inline-block;
+    height: auto;
+    margin: 0;
+    padding: 0 0 1px;
+    padding-inline-start: 36px;
+    position: relative;
+    min-height: 26px;
+    min-width: 100%;
+    text-align: start;
+    white-space: normal;
+    width: auto;
+
+    &::before {
+      inset-inline-start: 12px;
+      opacity: var(--doorhanger-icon-opacity);
+      top: 5px;
+    }
+
+    &:is(:hover, :focus-visible) {
+      background-color: var(--doorhanger-hover-bg-color);
+      color: var(--doorhanger-hover-color);
+    }
+
+    & > span {
+      display: block;
+      height: auto;
+      padding-inline-end: 4px;
+      width: auto;
```

Comment:
While it doesn't *seem* to break anything, I did notice when using the DOM inspector that these spans are now wider (previously the spans where only as big as the text, now they fill the entire button); was this change intentional?

Reply:
Good catch; it was not expected, but it should indeed not break anything. Nevertheless I found a simpler solution using `display: unset` so we don't need the width/height overrides at all anymore and the span automatically fits the content again.